docs/photutils/getting_started.rst

```diff
@@ -94,7 +94,7 @@ apertures:
     from photutils import datasets, daofind, aperture_photometry, CircularAperture
     hdu = datasets.load_star_image()
     image = hdu.data[500:700, 500:700]
-    image -= np.median(image)
+    image = image.astype(float) - np.median(image)
```

Comment:
I wonder whether it would clearer to leave this line as before and instead do ``image = hdu.data[...].astype(float)``